code/datums/special_traits/traits/traits.dm

```diff
@@ -560,12 +584,15 @@
 /datum/special_trait/reps_redemption
 	name = "Reps for Redemption"
 	greet_text = span_notice("Pain has finally transformed into gain.")
+	allowed_jobs = list(CHURCH_ROLES)
 	weight = 75
 
 /datum/special_trait/reps_redemption/on_apply(mob/living/carbon/human/character)
 	character.mind.adjust_skillrank(/datum/skill/misc/athletics, 1, TRUE)
-	character.change_stat("strength", 1)
-	character.change_stat("constitution", 1)
+	character.mind.adjust_skillrank(/datum/skill/combat/unarmed, 1, TRUE)
+	character.change_stat("strength", 2)
+	character.change_stat("constitution", 2)
+	character.change_stat("speed", -2)
```

Comment:
no, just no

Reply:
yes this is way too strong

Reply:
not just op buff, but limiting to church roles too

Reply:
It's called Reps for Redemption. Why would it **not** be a monastic body builder special?

Reply:
kets ask author @katsukai why he decided to use this word play

Reply:
If power lifting clergy is too 'OP', it can probably be reduced in weight from 70 to 50 or 25. 25 is associated with Legendary skills and hidden weapons however.

Reply:
A good way to ruin people's rounds smh. I get rerolling for the good specials is bad but also getting atrophy or bad week and needing to live with it through all your characters that week is so lame.

Reply:
Why cant you just make your own church special? Now you fucked this that people getting this special wont even know why it doesnt work because you didnt bother add a required text greeting to the player that they need to be church.